lib/active_merchant/billing/gateways/securion_pay.rb

```diff
@@ -0,0 +1,269 @@
+module ActiveMerchant #:nodoc:
+  module Billing #:nodoc:
+    class SecurionPayGateway < Gateway
+      self.test_url = 'https://api.securionpay.com/'
+      self.live_url = 'https://api.securionpay.com/'
+
+      self.supported_countries = Country::COUNTRIES.map { |country| country[:alpha2] }
```

Comment:
Please list out countries individually; this should reflect countries that merchants can get a SecurionPay account in.